shell/pages/c/_cluster/auth/user.retention/index.vue

```diff
@@ -0,0 +1,381 @@
+
+    return;
+  }
+
+  userRetentionSettings[SETTING.DISABLE_INACTIVE_USER_AFTER] = settings[SETTING.DISABLE_INACTIVE_USER_AFTER].value;
+});
+
+/**
+ * Watches the delete after period and removes the value if the checkbox is
+ * not selected. Lookup the value when the checkbox is selected.
+ */
+watch(deleteAfterPeriod, (newVal) => {
+  if (!newVal) {
+    userRetentionSettings[SETTING.DELETE_INACTIVE_USER_AFTER] = null;
+
+    return;
+  }
+
+  userRetentionSettings[SETTING.DELETE_INACTIVE_USER_AFTER] = settings[SETTING.DELETE_INACTIVE_USER_AFTER].value;
+});
+
+/**
+ * Watches both the disable and delete after periods. Clear all values in
+ * the form if both checkboxes are not selected. If one of the checkboxes
+ * are selected, lookup each of the values in the form.
+ */
+watch([disableAfterPeriod, deleteAfterPeriod], ([newDisableAfterPeriod, newDeleteAfterPeriod]) => {
+  if (!newDisableAfterPeriod && !newDeleteAfterPeriod) {
+    ids.forEach((key) => {
+      userRetentionSettings[key] = null;
+    });
+
+    validateUserRetentionCron();
+
+    return;
+  }
+
+  ids.filter((id) => ![SETTING.DISABLE_INACTIVE_USER_AFTER, SETTING.DELETE_INACTIVE_USER_AFTER].includes(id))
+    .forEach(assignSettings);
+
+  validateUserRetentionCron();
+});
+
+const assignSettings = (key: string) => {
+  if (settings[key].id === SETTING.USER_LAST_LOGIN_DEFAULT && settings[key].value && typeof settings[key].value === 'string') {
+    const value = settings[key].value as string;
+
+    userRetentionSettings[key] = dayjs(value).valueOf().toString();
+
+    return;
+  }
+
+  userRetentionSettings[key] = settings[key].value;
+};
+
+const fetchSetting = async(id: string) => {
+  return await store.dispatch('management/find', { type: MANAGEMENT.SETTING, id });
+};
+
+const ids = Object.keys(userRetentionSettings);
+const settingPromises = ids.map((id) => fetchSetting(id));
+
+onMounted(async() => {
+  settings = await Promise
+    .all(settingPromises)
+    .then((results) => results.reduce((acc, result) => {
+      return {
+        [result.id]: result,
+        ...acc,
+      };
+    }, { }));
+
+  ids.forEach(assignSettings);
+
+  disableAfterPeriod.value = !!userRetentionSettings[SETTING.DISABLE_INACTIVE_USER_AFTER];
+  deleteAfterPeriod.value = !!userRetentionSettings[SETTING.DELETE_INACTIVE_USER_AFTER];
+  loading.value = false;
+
+  validateUserRetentionCron();
+});
+
+const isFormValid = ref(false);
+const validateUserRetentionCron = () => {
+  const { [SETTING.USER_RETENTION_CRON]: cronSetting } = userRetentionSettings;
+
+  // Only require user retention cron when disable or delete after are active
+  if (!disableAfterPeriod.value && !deleteAfterPeriod.value) {
+    isFormValid.value = true;
+
+    return;
+  }
+
+  if (!cronSetting) {
+    isFormValid.value = false;
+
+    return;
+  }
+
+  if (typeof cronSetting === 'string' && !isValidCron(cronSetting)) {
+    isFormValid.value = false;
+
+    return;
```

Comment:
If you return an error message in this if block, you can use this function to show an error icon in the cron labeled input by passing the prop 
`:rules="[validateUserRetentionCron]"`

![Screenshot 2024-06-26 at 4 16 27 PM](https://github.com/rancher/dashboard/assets/42977925/89b55d3d-d3bb-4544-9e59-8019729e14ca)